src/MSBuild/XMake.cs

```diff
@@ -515,14 +516,7 @@ string commandLine
             ConsoleCancelEventHandler cancelHandler = Console_CancelKeyPress;
             try
             {
-#if (!STANDALONEBUILD)
-                // Enable CodeMarkers for MSBuild.exe
-                CodeMarkers.Instance.InitPerformanceDll(CodeMarkerApp.MSBUILDPERF,  String.Format(CultureInfo.InvariantCulture, @"Software\Microsoft\MSBuild\{0}", MSBuildConstants.CurrentProductVersion));
-#endif
-#if MSBUILDENABLEVSPROFILING 
-                string startMSBuildExe = String.Format(CultureInfo.CurrentCulture, "Running MSBuild.exe with command line {0}", commandLine);
-                DataCollection.CommentMarkProfile(8800, startMSBuildExe);
-#endif
+                MSBuildEventSource.Log.MSBuildExeStart();
```

Comment:
Since the command line can be passed as a single string or an array of strings, I had to wrap it in #if statements to determine which it was. Do you think that deserves a comment as to why I wrapped the array-of-strings case in IsEnabled?

Reply:
I think the different code is clear enough but would also be fine with a comment.